code/game/objects/structures/tables_racks.dm

```diff
@@ -41,12 +41,6 @@
 /obj/structure/table/proc/deconstruction_hints(mob/user)
 	to_chat(user, "<span class='notice'>The top is <b>screwed</b> on, but the main <b>bolts</b> are also visible.</span>")
 
-/obj/structure/table/Initialize()
-	. = ..()
-	for(var/obj/structure/table/T in src.loc)

```

Comment:
you'll need to update some lines below this, since they relied on this

Reply:
done